common/utils/thirdParty/gtag.js

```diff
@@ -24,7 +24,7 @@ const log = ({ methodName, ...rest }) => {
 const pageView = (url, isModalView = false) => {
   log({ methodName: 'pageview', url, isModalView });
 
-  if (isProduction) {
+  if (isProduction && window.gtag) {
```

Comment:
Can you add this for all the (isProduction) conditionals in this file? Even better might be: `isProduction && !!window && !!window.gtag`